kover-gradle-plugin/docs/aggregated.md

```diff
@@ -0,0 +1,67 @@
+# Kover Aggregated Plugin
+
+## Quickstart
+This plugin is a prototype for testing ideas related to an alternative configuration method.
+
+The main difference from the existing Kover Gradle Plugin is the reactive content of the report: only those classes and tests that were compiled and executed within the same build get into the report.
+
+To use the plugin, just add into a `settings.gradle.kts` file 
+```kotlin
+plugins {
+    id("org.jetbrains.kotlinx.kover.settings") version "0.8.1"
+}
+```
+**There is no need to apply Kover plugin in other places, the `org.jetbrains.kotlinx.kover` plug-in should not be applied anywhere.**
+
+To measure coverage you should pass special `-Pkover` argument to Gradle CLI command and call the command to generate the corresponding report `koverHtmlReport` or `koverXmlReport`.
+Example, if you want to measure the coverage of the `test` task:
+```shell
+./gradlew test -Pkover koverHtmlReport
+```
+
+Only those classes that were compiled as part of the current Gradle build are included in the report.
+If no compilation tasks were called in the build, the report will be empty.
+
+The report covers only those tests that were run as part of the current build.
+If no tests were called in the assembly, then the coverage for all classes will be 0.
+
+## Configuring
+At the moment, Kover Settings Plugin allows to configure reports minimally.
+
+There are two ways to configure, using a build script in `settings.gradle.kts` and CLI
+
+Acceptable settings in `settings.gradle.kts` and their equivalent in CLI
+```kotlin
+kover {
```

Comment:
If the end goal is moving this plugin to KGP, perhaps we should abstain from using `kover` as an extension name and use something less interesting, but at the same time also less obscure. For instance, `coverage` or `codeCoverage`. WDYT?

Reply:
So far, the purpose of this plugin is to collect feedback and test ideas for compatibility with Gradle.

Now we can't even guess what it will look like in KGP, what DSL it will be and what names will be used, because for this we need to connect additional teams.

It was enough for me so far that `kover {` and `-Pkover` write shorter than `coverage {` )